docs/source/markdown/podman-create.1.md

```diff
@@ -1224,6 +1225,19 @@ The default working directory for running binaries within a container is the roo
 The image developer can set a different default with the WORKDIR instruction. The operator
 can override the working directory by using the **-w** option.
 
+#### **\-\-pidfile**=*path*
+
+When the pidfile location is specified, the container process' PID will be written to the pidfile. (This option is not available with the remote Podman client)
+After the container is started, the location for the pidfile can be discovered with the following `podman inspect` command:
+
+    $ podman inspect --format '{{ .Config.CreateCommand }}' $CID
+    podman run --pidfile $pidfilepath image
+
+If the pidfile option is not specified, the container process' PID will be written to
+/run/containers/storage/${storage-driver}-containers/$CID/userdata/pidfile.
+The default pidfile location is not listed in the `podman inspect` output.
```

Comment:
No we need you to add the pid file location to the inspect.  Just copy the way that conmonpidfile is handled.

Reply:
Sorry for misunderstanding.